File: backend/src/main/scala/ch/countrygame/Classes/Modifier.scala
package ch.countrygame.Classes

import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}

case class Modifier(modifierId: Int, modifierName: String, modifierValue: Double) {
}

object Modifier {
  implicit val encoder = deriveEncoder[Modifier]
  implicit val decoder = deriveDecoder[Modifier]
}

File: frontend/src/app/models/modifier.ts
export class Modifier {
  modifierId: number;
  modifierName: string;
  modifierValue: number;

  constructor(modifierId: number, modifierName: string, modifierValue: number) {
    this.modifierId = modifierId;
    this.modifierName = modifierName;
    this.modifierValue = modifierValue;
  }
}
